sceptre/context.py

```diff
@@ -40,7 +43,7 @@ class SceptreContext(object):
     :type no_colour: bool
     """
 
-    def __init__(self, project_path, command_path,
+    def __init__(self, project_path, command_path, config_path="config",
```

Comment:
Agreed. I've changed the name of this parameter to `config_directory` as requested. 

I've also updated docs for the cli, and added an example under `architecture.md` of how this might change your project structure. This is particularly useful if you have several templates you want to share among projects  (e.g. `vpc.yaml`) to ensure consistency of deployment.